pymatviz/correlation.py

```diff
@@ -43,7 +43,7 @@ def marchenko_pastur_pdf(x: float, gamma: float, sigma: float = 1) -> float:
 
 
 def marchenko_pastur(
-    matrix: ArrayLike,
+    matrix: NDArray,
```

Comment:
i think `ArrayLike` is actually more accurate since `np.linalg.eigvalsh(matrix)` also handles nested lists